gradle/verification-metadata.xml

```diff
@@ -242,6 +258,91 @@
+            <sha256 value="4689bc902255a19fe98277683ba3231c094d107c54c8d35f2b6f9c97d226418e" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-jupiter-engine-5.8.1.module">
+            <sha256 value="68790fec33f97b0ec840cf47ac40ee0c72e056f122ca0f3c6646743349937699" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="org.junit.jupiter" name="junit-jupiter-params" version="5.8.1">
+         <artifact name="junit-jupiter-params-5.8.1.jar">
+            <sha256 value="389b8d13a8d8872fcbd4f0eba7b2c46afc628419f9a1b2a3a9f93241a06a7218" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-jupiter-params-5.8.1.module">
+            <sha256 value="124d603c6d8033366d8ca471636b446e38b9cc1bd010f329542358187afa865e" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="org.junit.platform" name="junit-platform-commons" version="1.8.1">
+         <artifact name="junit-platform-commons-1.8.1.jar">
+            <sha256 value="fa4fa68c8bd54dd0cb49c3fcbe9b2e42f4da6bedbe7e7ccf2a05f1a1e609b593" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-platform-commons-1.8.1.module">
+            <sha256 value="698fd0541acb7effc6659848fd0c7dd5010a49f15f0c1cba43e535807f90f66b" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="org.junit.platform" name="junit-platform-engine" version="1.8.1">
+         <artifact name="junit-platform-engine-1.8.1.jar">
+            <sha256 value="702868ed7e86b9b4672ede0f1e185e905baca9afab57746a7c650be3c7bca047" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-platform-engine-1.8.1.module">
+            <sha256 value="d9f420a64539a3edf60f80df0c6fd722b750725744c79ca40f7d25ae56ca4ba4" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="org.junit.vintage" name="junit-vintage-engine" version="5.8.1">
+         <artifact name="junit-vintage-engine-5.8.1.jar">
+            <sha256 value="176b53cd1bdbf9233e96c8b0c7a9e5b864282fbbdeba43b5ceafdedb29a49156" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="junit-vintage-engine-5.8.1.module">
+            <sha256 value="9ce9fa2e4ee6a740d610102530462a71ce0fa9d2f1418522e4b2bdb6072f679a" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="org.opentest4j" name="opentest4j" version="1.2.0">
+         <artifact name="opentest4j-1.2.0.jar">
+            <sha256 value="58812de60898d976fb81ef3b62da05c6604c18fd4a249f5044282479fc286af2" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="opentest4j-1.2.0.pom">
+            <sha256 value="a96e671816c1ff8803bdec74c9241f025bdfb277da5d2b4ee02266405936f994" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
Is `opentest4j` actually required in order to support JUnit 5? If not, this should be reverted. If so, that's another third-party dependency we would need to get approval for.

Reply:
Will try block

Reply:
I couldn't block opentest4j.  Apparently this is a project that harmonizes the exceptions thrown by various test frameworks (not just JUnit 5) and JUnit 5 depends on it for the exceptions it uses to signal test failures and other error conditions.  It is a fairly small project (6 classes only) licensed under the Apache 2.0 license: https://github.com/ota4j-team/opentest4j

Reply:
Since we only use JUnit for build / test, and don't directly depend on `opentest4j`, this is not a problem.